src/Sentry.DiagnosticSource/Internal/DiagnosticSource/EFCommandDiagnosticSourceHelper.cs

```diff
@@ -0,0 +1,48 @@
+using Sentry.Internal.Extensions;
+
+namespace Sentry.Internal.DiagnosticSource;
+
+// ReSharper disable once InconsistentNaming
```

Comment:
I don't think these ReSharper comments are needed.  They flag as unneeded in Rider on my machine anyway.  Maybe just update abbreviations locally if they're flagging for you.  (same on other classes.)

Reply:
Hm, no idea how to configure that. I'll remove the comments and just learn to live with the squiggly lines I think.